quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/LoggingPropertyMappers.java

```diff
@@ -192,6 +197,15 @@ private static void setCategoryLevel(String category, String level) {
 
     record CategoryLevel(String category, String levelName) {}
 
+    private static void validateLogParameters(String level) {
+        try {
+            toLevel(level);
+        } catch (IllegalArgumentException iae) {
+            throw new PropertyException(
+                    String.format("Invalid parameter: %s", level));
```

Comment:
This is not descriptive enough. It should mimic the previous error message and possibly mention the level is not case-sensitive.

Reply:
Hi @shawkins ,yes, but in the function validateLogParameters , we are unable to get the option paramas like `--log-console-level` . Also, do you want me to hardcode these options `off, fatal, error, warn, info, debug, trace, all` ?

Reply:
> we are unable to get the option params like --log-console-level

That is accessible via addValidator - but since that is additive, you'd first have to call validator with something that does nothing. Alternatively, you can have a method that returns a function:

```
private static Consumer<String> validateLogParameters(String param, String level) {
   ...
```

> Also, do you want me to hardcode these options

No, use Stream.of(Level.values()).collect(Collectors.joining(", "))

Reply:
Hi @shawkins , yes I have implemented the suggested changes, thanks !